yamls/jp/overlays.yaml

```diff
@@ -1144,17 +1144,17 @@
     compress: True
     bss_size: 0x380
     subsegments:
-      - [0x8253C0, asm, overlays/other_ovl/ovl__008253C0/ovl__008253C0]
-      - [0x8267B0, data, overlays/other_ovl/ovl__008253C0/ovl__008253C0]
-      - { start: 0x827BE0, type: bss, vram: 0x809303E0, name: overlays/other_ovl/ovl__008253C0/ovl__008253C0 }
+      - [0x8253C0, asm, overlays/managers/ovl__008253C0/ovl__008253C0]
```

Comment:
Why you didn't change this one to `c` too?

Reply:
forgor, I was too eepy